src/assets/panties/crotch_jute_rope/crotch_jute_rope.asset.ts

```diff
@@ -0,0 +1,158 @@
+DefineAsset({
+	name: 'Crotch Jute Ropes',
+	size: 'small',
+	graphics: 'graphics.json',
+	colorization: {
+		rope: {
+			name: 'Rope',
+			default: '#D7AC4D',
+		},
+	},
+	attributes: {
+		provides: [
+			'Restraint',
+			'Restraint_legs',
+			'Chastity',
+		],
+		requires: [
+			'!Penis',
+		],
+	},
+	// size:260, y:560, centered
+	preview: 'preview.png',
+	modules: {
+		waist: {
+			type: 'typed',
+			name: 'Waist',
+			variants: [
+				{
+					id: 'none',
+					name: 'No',
+					default: true,
+				},
+				{
+					id: 'rope',
+					name: 'Rope',
+					properties: {
+						attributes: {
+							provides: [
+								'Waist_rope_anchor_point',
+							],
+						},
+					},
+				},
+			],
+		},
+		crotch: {
+			type: 'typed',
+			name: 'Crotch',
+			variants: [
+				{
+					id: 'open',
+					name: 'Open',
+
+				},
+				{
+					id: 'rope',
+					name: 'Rope',
+					default: true,
+					properties: {
+						attributes: {
+							provides: [
+								'Vulva_cover',
+								'Vulva_rope_anchor_point',
+							],
+							covers: [
+								'Vulva_item',
+							],
+						},
+					},
+				},
+				{
+					id: 'knot',
+					name: 'Knot',
+					properties: {
+						attributes: {
+							provides: [
+								'Vulva_cover',
+								'Vulva_rope_anchor_point',
+							],
+							covers: [
+								'Vulva_item',
+							],
+						},
+					},
+				},
+			],
+		},
+		hips: {
+			type: 'typed',
+			name: 'Hips',
+			variants: [
+				{
+					id: 'none',
+					name: 'None',
+					default: true,
+					properties: {
+						stateFlags: {
+							requires: {
+								thigs_free: 'Rope over hips not tied to thighs',
```

Comment:
Nitpick: Depending on how other requirements are updated, it might be nice to reword for consistency.